integration-tests/src/test/java/tachyon/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -510,6 +510,10 @@ public void deleteRootTest() throws Exception {
   public void getCapacityBytesTest() {
     BlockMaster blockMaster =
         mLocalTachyonClusterResource.get().getMaster().getInternalMaster().getBlockMaster();
+    // Sleep to give the workers time to register with the master
+    // TODO(andrew): Remove this when mLocalTachyonClusterResource.start() blocks until workers have
+    // registered with master.
+    CommonUtils.sleepMs(200);
```

Comment:
Can we instead add logic that repeatedly checks if the worker has registered with the master instead of relying on an ad hoc sleep interval?